### PR TITLE
Remove print statements, remove r9k mode, let users with MOD privilege move threads

### LINKED ISSUE #34: disable r9k mode
Allow posters to post duplicate attachments and/or images. This doesn't seem to be a setting (that we can find) but actually hardcoded

### LINKED ISSUE #45: Mods can move threads
Add this to instance-config
`$config['mod']['move'] = MOD;`

### PR DESCRIPTION
**Pull request by towards-a-new-leftypol**
https://github.com/towards-a-new-leftypol/leftypol_lainchan/issues/34
https://github.com/towards-a-new-leftypol/leftypol_lainchan/issues/45

### CODE REVIEW

File: inc/functions.php [hunk 1]
```diff
@@ -2299,11 +2275,10 @@ function strip_combining_chars($str) {
 function buildThread($id, $return = false, $mod = false) {
 	global $board, $config, $build_pages;
 	$id = round($id);
-    print_err("buildThread start");
 
-	if (event('build-thread', $id)) {
-		return;
-    }
+  if (event('build-thread', $id)) {
```
Comment (towards-a-new-leftypol) [inline inc/functions.php:2279]: this was just formatting

**Review by nonmakina — APPROVED** [review]
Looks good.

File: inc/functions.php (hunk 1 above)
Comment (nonmakina) [reply inc/functions.php:2279]: The indent looks inconsistent from here.

Comment (towards-a-new-leftypol) [reply inc/functions.php:2279]: Yes it was a tabs vs spaces issue. I'm probably going to start converting things to 4 spaces. This has been fixed on myownbranch but I only saw your comment after I already merged.

**Pull request merged**